media-video/ffmpeg/ffmpeg-4.2.2.recipe

```diff
@@ -158,9 +185,33 @@ BUILD_REQUIRES="
 	"
 if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
 	BUILD_REQUIRES+="
+		devel:libass$secondaryArchSuffix
+#		devel:libbluray$secondaryArchSuffix
+#		devel:libbs2b$secondaryArchSuffix
+#		devel:libcaca$secondaryArchSuffix
 		devel:libdav1d$secondaryArchSuffix
+		devel:libfdk_aac$secondaryArchSuffix
+		devel:libfontconfig$secondaryArchSuffix
+		devel:libfreetype$secondaryArchSuffix
+		devel:libfribidi$secondaryArchSuffix
+		devel:libgme$secondaryArchSuffix
+		devel:libgmp$secondaryArchSuffix
+		devel:libopenjp2$secondaryArchSuffix
 		devel:libopenmpt$secondaryArchSuffix
+#		devel:librsvg_2$secondaryArchSuffix
```

Comment:
Since I did the update for librsvg, did you check why the build needs/fails on it?

Reply:
I just don't see the point enabling librsvg for ffmpeg. Is it really a use case to open svg files with MediaPlayer? Or to save a bitmap image (because factually ffmpeg gives a non-vctor output) to svg with ffmpeg? Or is there any real-world use case for that?
We could enable tessaract or tensorflow, or many other things (ok, we don't have tensorflow port yet), but we need to check f that makes any sense.
Other argumentation coulkd be: it is possible to do something, so it must be allowed to do that. I can accept that kind of argumentation, but i still don't see the point.

Reply:
Didn't check on how ffmpeg uses it, just asking :)

Reply:
FWIW, you can use the commandline "ffmpeg" to convert images from svg to for instance png, you can join them as a group into a video file that can be played with MP (last check I did however was so fast you couldn't see the starting images, but there's an option for delay IIRC) :)